lib/scope.js

```diff
@@ -235,6 +235,7 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
                     ref.reference(options);
                 });
                 node.thedef = def;
+                node.reference(options);
```

Comment:
The AST node is referencing mangle *options*?!!

This definitely needs to commented.